lib/matplotlib/tri/tritools.py

```diff
@@ -12,18 +12,13 @@ class TriAnalyzer:
     """
     Define basic tools for triangular mesh analysis and improvement.
 
-    A TriAnalyzer encapsulates a :class:`~matplotlib.tri.Triangulation`
-    object and provides basic tools for mesh analysis and mesh improvement.
+    A TriAnalyzer encapsulates a `~matplotlib.tri.Triangulation` object and
+    provides basic tools for mesh analysis and mesh improvement.
 
     Parameters
     ----------
-    triangulation : :class:`~matplotlib.tri.Triangulation` object
+    triangulation : `~matplotlib.tri.Triangulation`
         The encapsulated triangulation to analyze.
-
-    Attributes
-    ----------
-    `scale_factors`
```

Comment:
would be nice if we could get sphinx to like :attr: to here...

Reply:
If not using markup, this links to the property documentation.

I'm not sure if we should list properties as attributes if they are not writable, but attribute handling is for another time. With this change this is already an improvement.

Reply:
ah, that works nicely